### PR TITLE
VM name limited to 23 char

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1857181

CP for 4.5, 4.6

### CODE REVIEW

File: modules/virt-importing-vm-cli.adoc [hunk 1]
```diff
@@ -119,7 +119,7 @@ spec:
 EOF
 ----
 <1> If you create a ResourceMapping CR, uncomment the `resourceMapping` section.
-<2> The `targetVmName` must not exceed 23 characters in length.
+<2> The `targetVmName` is limited to 23 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
```
Comment (kalexand-rh) [inline modules/virt-importing-vm-cli.adoc:122]: s/is limited/value is limited

**Review by kalexand-rh — COMMENTED** [review]
One pick, and then it LGTM

File: modules/virt-importing-vm-cli.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-importing-vm-cli.adoc]: Done

**Pull request merged**